src/test_suite/test_runner_main.ts

```diff
@@ -162,155 +203,208 @@ export async function testRunnerMain(
-    );
-    console.log(`                       (default is -v=${defaultProcessor}).`);
-    console.log('-x                     Do not verify intermediate results.');
-    console.log('-d <datapath>          Path to prix-fixe data files.');
-    console.log('                           attributes.yaml');
-    console.log('                           intents.yaml');
-    console.log('                           options.yaml');
-    console.log('                           products.yaml');
-    console.log('                           quantifiers.yaml');
-    console.log('                           rules.yaml');
-    console.log('                           stopwords.yaml');
-    console.log('                           units.yaml');
-    console.log(
-        '                       The -d flag overrides the value specified'
-    );
-    console.log(
-        '                       in the PRIX_FIXE_DATA environment variable.'
-    );
-    console.log('-h|help|?              Show this message.');
-    console.log('-s suite               Run tests in specified suite.');
-    console.log(
-        '-t <raw|stt|scoped>  Run tests using specified utterance field. RAW | STT | SCOPED'
-    );
-    console.log(
-        '                           The default is SCOPED and will use the highest corrected value provided in the yaml.'
-    );
-    console.log(
-        '                           RAW: force it to run on the original input, even if there are corrected versions available'
-    );
-    console.log(
-        '                             STT: if there is correctedSTT available use that otherwise use input, even if there is correctedScope available'
-    );
-    console.log(
-        '                          SCOPED: if there is correctedScope available use it, other wise fall back to correctedSTT and then input'
-    );
-    console.log(' ');
+            console.log('');
+            console.log('');
 
-    if (processorFactory.count() > 0) {
-        console.log('Available Processors:');
-        for (const processor of processorFactory.processors()) {
-            console.log(`  "-v=${processor.name}": ${processor.description}`);
-            // TODO: list expected data files for each processor.
+            suiteResults.results.forEach(r => allResults.recordResult(r));
         }
-    } else {
-        console.log('No Processors available.');
+
+        const testPassRate =
+            (allResults.passCount /
+                (allResults.passCount + allResults.failCount)) *
+            100;
+
+        console.log();
+        console.log('============================');
+        console.log(
+            `OVERALL RESULTS: ${allResults.passCount}/${allResults.results.length}`
+        );
         console.log(
-            'The supplied ProcessorFactory has no ProcessorDescriptions'
+            chalk`${allResults.results.length.toString()} tests run; {yellow.bold ${testPassRate.toString()}%} pass rate.`
```

Comment:
We brought `chalk` in, but we're only using it here. I think we should look to either apply it to more output or remove it from here and, thus, as an overall dependency, because it's not providing enough value for the costs.

Reply:
Also, we already had ansi-styles (lower level API which chalk uses). We should standardize on one or the other.

We should also ask whether introducing colors hurts us when grepping logs. @drew: do you have experiences (positive or negative) with the introduction of colors in console spew?

Reply:
I'll remove chalk then since it's so minimal atm; I don't think I want to put in effort to update all the console.logs